org.lamport.tla.toolbox.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
```

Comment:
This should be contributed upstream.

Reply:
What do you mean with this?

Reply:
@jesmg the desktop file should be at the upstream sources, as the metainfo.

The goal is to keep at flathub repo the less the information as possible. Ideally just the manifest.

Reply:
Please follow it up with @lemmy after the merge.